### PR TITLE
[LT-534]  Added default file read code for all languages and option to enable in studio

### PR DESCRIPTION
**Pull request by rehanedly**

### CODE REVIEW

File: openassessment/assessment/migrations/0004_auto_20220411_2015.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    dependencies = [
+        ('assessment', '0003_expand_course_id'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='assessment',
+            name='feedback',
+            field=models.TextField(blank=True, default='', max_length=10000),
+        ),
```
Comment (HussainTaj-arbisoft) [inline openassessment/assessment/migrations/0004_auto_20220411_2015.py:17]: I think we should remove this.

File: openassessment/workflow/migrations/0003_auto_20220411_2015.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workflow', '0002_remove_django_extensions'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```
Comment (HussainTaj-arbisoft) [inline openassessment/workflow/migrations/0003_auto_20220411_2015.py:14]: I think we should remove this.

File: openassessment/workflow/migrations/0004_auto_20220411_2017.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workflow', '0003_auto_20220411_2015'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```
Comment (HussainTaj-arbisoft) [inline openassessment/workflow/migrations/0004_auto_20220411_2017.py:14]: I think we should remove this.

**Review by HussainTaj-arbisoft — CHANGES_REQUESTED** [review]
1. Shouldn't the base of this PR be `litmustest.develop` instead of `litmustest.staging`?
2. This PR includes migrations of fields we did not change. I think we should not do this on our side because the original (newer) codebase would have the new fields. If we need the migrations we should update the repos/packages.
3. We need to run the following commands to generate a minified buddle of the JS, otherwise this won't work on the server.

```bash
make lms-shell
cd /edx/src/edx-ora2-1/
make javascript
```

**Comment by rehanedly** [comment]
> 1. Shouldn't the base of this PR be `litmustest.develop` instead of `litmustest.staging`?
> 2. This PR includes migrations of fields we did not change. I think we should not do this on our side because the original (newer) codebase would have the new fields. If we need the migrations we should update the repos/packages.
> 3. We need to run the following commands to generate a minified buddle of the JS, otherwise this won't work on the server.
> 
> ```shell
> make lms-shell
> cd /edx/src/edx-ora2-1/
> make javascript
> ```

Fixed these issues

**Review by HussainTaj-arbisoft — APPROVED** [review]
LGTM 👍

**Pull request merged**